tutorials/beginners-cookbook/index.md

```diff
@@ -392,7 +426,7 @@ var featNew = feature.select(['name'], ['descriptor']);
 var featVal = feature.get('size');
 ```
 
-#### Example: Feature operations
+#### Example: Feature operations [[Open In Code Editor](https://code.earthengine.google.com/043cc41ccb2332494a7d0fbb2bc4d5bc)]
```

Comment:
Please don't exclude Code Editor links.

Reply:
Sorry, an unfortunate typo. I meant to write "please don't _include_ them" (i.e. leave them out)

Reply:
Removed the link to the code editor.